app/page.tsx

```
@@ -431,50 +461,63 @@ export default function Home() {
+                          </div>
+                          <div className="text-sm font-medium text-white/60">
+                            {Math.round(match.similarity * 100)}% Match
+                          </div>
                         </div>
-                        <p className='text-sm text-white/60'>{match.reason}</p>
+                        <p className="text-sm text-white/60">
+                          {match.matchreason}
+                        </p>
+                        <ul
+                          className="text-sm text-white/60 list-disc list-inside space-y-1"
+                          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)